### ISSUE TITLE
Readme file needs to be updated

### ISSUE THREAD

**Issue by dumbcoder7** [mate-academy/layout_stars#292]
The DEMO LINK field in README doesn't work even after inputting username. Perhaps, an example could do the trick?
There are also some grammatical errors in the file. If it is fine, I can take care of the issue.

**Comment by NikolayButylkov** [comment]
@adityaraute Yes, thanks for the feedback! Problem with DEMO LINK DEMO LINK was from cache. I reload page(hard reload) with shift + alt + update (and I think it will be work)
https://prnt.sc/10vcnut (PrntScr)
